content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -57,15 +57,14 @@ Choose the instructions corresponding to your environment:
 
 {{< tab name="external load balancer" category-value="external-lb" >}}
 
-Follow these instructions if you have determined that your environment has an external load balancer.
+**Follow these instructions if you have determined that your environment has an external load balancer.**
 
 Set the ingress IP and ports:
 
 {{< text bash >}}
 $ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
 $ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].port}')
 $ export SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="https")].port}')
-$ export TCP_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="tcp")].port}')
```

Comment:
Why are you deleting TCP_INGRESS_PORT, it's used in other docs

Reply:
Added it back.